packages/chatbot-server-mongodb-public/.env.example

```diff
@@ -5,7 +5,7 @@ OPENAI_EMBEDDING_DEPLOYMENT=<deployment_name>
 OPENAI_EMBEDDING_MODEL=text-embedding-ada-002
 OPENAI_EMBEDDING_MODEL_VERSION=2023-03-15-preview
 OPENAI_CHAT_COMPLETION_DEPLOYMENT=<deployment_name>
-OPENAI_CHAT_COMPLETION_MODEL_VERSION=<model_version>
+OPENAI_CHAT_COMPLETION_MODEL_VERSION=2023-05-15
```

Comment:
Intended?

Reply:
yea

Reply:
oh this should have been in mongodb-chatbot-server/.env.example, though can also be here. will add there too.